az-core/src/main/java/azkaban/Constants.java

```diff
@@ -177,4 +177,17 @@
     public static final String JOBCALLBACK_RESPONSE_WAIT_TIMEOUT = "jobcallback.response.wait.timeout";
     public static final String JOBCALLBACK_THREAD_POOL_SIZE = "jobcallback.thread.pool.size";
   }
+
+  public static class DependencyProperties {
+
+    public static final String DEPENDENCY_NAME = "dependency.name";
```

Comment:
Do we need to specify the the prefix like dependency.* or schedule.* ? In the yaml file, I'm expecting it to look like this:
```
schedule:                    
    type: cron
    value: 0 0 1 ? * * 
    max.wait.time:  …. 
  dependencies:           
    name: search-impression
    type: dali-dataset
    params:    
        view : search_mp_versioned.search_impression_event_0_0_47 
        delay :1
```

Reply:
i think you are right.  name and type will be simpler.